app/gui2/src/stores/suggestionDatabase/index.ts

```diff
@@ -85,7 +86,20 @@ class Synchronizer {
 export const useSuggestionDbStore = defineStore('suggestionDatabase', () => {
   const entries = reactive(new SuggestionDb())
   const groups = ref<Group[]>([])
+  const methodPointerToEntry = reactive(new Map<string, Map<string, SuggestionEntry>>())
+
+  watchEffect(() => {
```

Comment:
This watcher will be triggered on every database modification, making it really inefficient. Temporarily this is OK, since we will soon have reactive database indexing (currently being worked on by @vitvakatu). Once it's in, this code will have to be updated to make use of it. Please leave a FIXME comment to not miss it later.
```suggestion
  // FIXME: Replace this inefficient watcher with reactive index, once we have it developed.
  watchEffect(() => {
```